### PR TITLE
0.4.0 revised documentation

### PR DESCRIPTION
**Pull request by talonchandler**
This PR revises `recOrder`'s documentation pages to match current behavior.

### CODE REVIEW

File: docs/napari-plugin-guide.md [hunk 1]
```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
...
+## Connect to MicroManager
+Click “Connect to MM”. If the connection succeeds, proceed to calibration. If not, revisit the [microscope installation guide](./microscope-installation-guide.md).
+
+![](./images/connect_to_mm.png)
 
 ## Calibration tab
-The first step in the QLIPP process is to calibrate the liquid crystals. This process involves generating the polarization states and acquiring a background **on an empty FOV**, so begin by placing your sample on the stage and focusing on the surface of the coverslip or well. 
 
-The light path must also be in **Kohler Illumination** to ensure uniform illumination of the sample. [Please follow these steps to setup Kohler illumination.](https://www.microscopyu.com/tutorials/kohler)
+The first step in the acquisition process is to calibrate the liquid crystals and measure a background. In the `recOrder` plugin you will see the following options for controlling the calibration:
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:21]: Should we note that this is only necessary for polarization imaging?

File: docs/napari-plugin-guide.md [hunk 2]
```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
...
+
+![](./images/run_calib.png)
+
 
-After pressing “Connect to MM” and choosing a directory (where the calibration metadata / background images will be stored), the first step in calibration is to input the illumination wavelength and decide on the swing to use. 
+### Prepare for a calibration
+Place your sample on the stage, focus on the surface of the coverlip/well, navigate to **an empty FOV**, then align the light source into **Kohler illumination** [following these steps.](https://www.microscopyu.com/tutorials/kohler)
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:27]:
```suggestion
Place your sample on the stage, focus on the surface of the coverslip/well, navigate to **an empty FOV**, then align the light source into **Kohler illumination** [following these steps](https://www.microscopyu.com/tutorials/kohler).
```

File: docs/napari-plugin-guide.md [hunk 3]
```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
...
 
-The plot shows the intensities over time during the optimization. One way to diagnose if calibration is going smoothly is to look at the shape of this plot. An example of an ideal plot is below:
+The plot shows the intensities over time during calibration. One way to diagnose an in-progress calibration is to watch the intensity plot. An ideal plot will look similar to the following:
 
-![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/ideal_plot.png)
+![](./images/ideal_plot.png)
 
-Once finished, you will get a calibration assessment and an extinction value. The calibration assessment will tell you if anything is incorrect with the light path or if calibration went awry. Extinctions gives you a metric for calibration quality—the higher the extinction, the cleaner the light path and the greater the sensitivity of QLIPP.
+Once finished, you will get a calibration assessment and an extinction value. The extinction value  gives you a metric for calibration quality: the higher the extinction, the cleaner the light path and the greater the sensitivity of QLIPP.
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:54]:
```suggestion
Once finished, you will get a calibration assessment and an extinction value. The extinction value gives you a metric for calibration quality: the higher the extinction, the cleaner the light path and the greater the sensitivity of QLIPP.
```

File: docs/napari-plugin-guide.md [hunk 4]
```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
...
+* **Extinction 0 – 50**:  Very poor. The alignment of the universal compensator may be off or the sample chamber may be highly birefringent. 
 
-* Extinction 50-100: Okay extinction, could be okay for tissue imaging and strong anisotropic structures. Most likely not suitable for cell imaging
+* **Extinction 50 - 100**: Okay extinction, could be okay for tissue imaging and strong anisotropic structures. Most likely not suitable for cell imaging
 
-* Extinction 100-200: Good Extinction. These are the typical values we get on our microscopes.
+* **Extinction 100 - 200**: Good Extinction. These are the typical values we get on our microscopes.
 
-* Extinction 200+: Excellent. Indicates a very well-aligned and clean light path and high sensitivity of the system.
+* **Extinction 200**+: Excellent. Indicates a very well-aligned and clean light path and high sensitivity of the system.
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:62]:
```suggestion
* **Extinction 200+**: Excellent. Indicates a very well-aligned and clean light path and high sensitivity of the system.
```

File: docs/napari-plugin-guide.md [hunk 5]
```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
 ```
 napari -w recOrder-napari
 ```
+## Connect to MicroManager
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:14]: Not a big deal, but the official spelling seems to be [Micro-Manager](https://micro-manager.org/Micro-Manager_User's_Guide).

File: docs/napari-plugin-guide.md [hunk 6]
```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
...
 ### Acquisition Settings
-*Z Start, Z End, Z Step* specify the relative z-parameters to use for acquiring an image volume. Values are in the default units of the stage, typically in microns. The center slice will be the current position of the z-stage
+![](./images/acquisition_settings.png)
+
+The **Acquisition Mode** sets the target dimensions for the reconstruction. Perhaps surprisingly, all 2D reconstructions require 3D data except for **Retardance + Orientation** in **2D Acquisition Mode**. The following table summarizes the data that will be acquired when an acquisition button is pressed in **2D** and **3D** acquisition modes:
 
-![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/acquisition.png)
+| **Acquisition** \ Acquisition Mode | 2D mode | 3D mode |  
+| :--- | :--- | :--- |
+| **Retardance + Orientation** | CYX data | CYX data |
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:104]: The previous paragraph says:

> The following table summarizes the data that will be acquired

So in the case of 3D birefringence, although the reconstruction only sees a CYX at a time, CZYX will still be acquired (and reconstructed).

File: docs/napari-plugin-guide.md [hunk 7]
```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
...
 
-![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/acquisition.png)
+| **Acquisition** \ Acquisition Mode | 2D mode | 3D mode |  
+| :--- | :--- | :--- |
+| **Retardance + Orientation** | CYX data | CYX data | 
+| **Phase From BF** | ZYX data | ZYX data | 
+| **Retardance + Orientation + Phase** | CZYX data | CZYX data | 
 
-Ex. for a 20 um thick cell, the user would first focus in the middle of the cell and then set the following parameters: 
+Unless a **Retardance + Orientation** reconstruction in **2D Acquisition Mode** is requested, `recOrder` uses MicroManager's z-stage to acquire 3D data. **Z Start**, **Z End**, and **Z Step** are stage settings for acquiring an image volume, relative to the current position of the stage. Values are in the stage's default units, typically in microns.
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:108]:
```suggestion
Unless a **Retardance + Orientation** reconstruction in **2D Acquisition Mode** is requested, `recOrder` uses MicroManager's z-stage to acquire 3D data. **Z Start**, **Z End**, and **Z Step** are stage settings for acquiring an image volume, relative to the current position of the stage. Values are in the stage's default units, typically in micrometers.
```

File: docs/napari-plugin-guide.md [hunk 8]
```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
...
-* Condenser NA: Numerical Aperture of Condenser
-* Magnification: Magnfication of the objective
-* Camera Pixel Size: Pixel size of the camera in microns (ex. 6.5)
-* RI of Obj Media: Refractive index of the objective media. Defaults to air (1.003). Typical values also include 1.512 (oil) or 1.473 (glycerol).
-* Z Padding: The number of slices to pad on either end of the stack in order to correct for edge reflection artifacts. Necessary if the sample is not fully out of focus on either end of the stack.
+* **GPU ID**: Not implemented
+* **Wavelength (nm)**: illumination wavelength
+* **Objective NA**: numerical aperture of the objective, typically found next to magnification
+* **Condenser NA**: numerical aperture of the condenser
+* **Camera Pixel Size (um)**: pixel size of the camera in microns (e.g. 6.5 um)
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:135]:
```suggestion
* **Camera Pixel Size (um)**: pixel size of the camera in micrometers (e.g. 6.5 μm)
```

File: docs/napari-plugin-guide.md [hunk 9]
```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
...
+* **RI of Obj. Media**: refractive index of the objective media, typical values are 1.0 (air), 1.3 (water), 1.473 (glycerol), or 1.512 (oil)
+* **Magnification**: magnification of the objective
+* **Rotate Orientation (90 deg)**: rotates "Orientation" reconstructions by +90 degrees clockwise and saves the result, most useful when a known-orientation sample is available
+* **Flip Orientation (90 deg)**: flips "Orientation" reconstructions about napari's horizontal axis and saves the result
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:139]:
```suggestion
* **Flip Orientation**: flips "Orientation" reconstructions about napari's horizontal axis before saving the result
```

File: docs/napari-plugin-guide.md (hunk 1 above)
Comment (talonchandler) [reply docs/napari-plugin-guide.md:24]: Good call, fixed.

File: docs/napari-plugin-guide.md [hunk 10]
```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
...
+* **Invert Phase Contrast**: inverts the phase reconstruction's contrast by flipping the positive and negative directions of the stage during the reconstruction, and saves the result 
+
+### Phase Reconstruction Settings
+![](./images/phase_reconstruction_settings.png)
+
+These parameters are used only by phase reconstructions
 
-The acquired data will then be displayed in the `napari` window. Note that phase reconstruction is more computationally expensive and may take several minutes depending on your system.
+* **Z Padding**: The number of slices to pad on either end of the stack in order to correct for edge reflection artifacts, necessary if the sample is not fully out of focus on either end of the stack
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:150]: Is this reflection padding?

File: docs/napari-plugin-guide.md (hunk 5 above)
Comment (talonchandler) [reply docs/napari-plugin-guide.md]: Fixed. Thanks!

File: docs/napari-plugin-guide.md (hunk 6 above)
Comment (talonchandler) [reply docs/napari-plugin-guide.md]: That was an error! Good eye. Retardance + Orientation 3D should be CZYX.

Comment (talonchandler) [reply docs/napari-plugin-guide.md]: Fixed

File: docs/napari-plugin-guide.md (hunk 10 above)
Comment (talonchandler) [reply docs/napari-plugin-guide.md]: Not quite. Simplified.

File: docs/napari-plugin-guide.md (hunk 7 above)
Comment (talonchandler) [reply docs/napari-plugin-guide.md]: Thanks, changed all "microns" to "micrometers".

**Review by ziw-liu — APPROVED** [review]

**Pull request merged**